compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -51,11 +51,11 @@ genericBinaryEvaluator(TR::Node *node, TR::InstOpCode::Mnemonic regOp, TR::InstO
    TR::Register *trgReg = NULL;
    int64_t value = 0;
 
-   if(1 == firstChild->getReferenceCount())
+   if(true == firstChild->isSingleRef())
```

Comment:
These seem redundant. `isSingleRef` returns a boolean value already, so the `true ==` is unnecessary IMO. Could we remove it throughout this PR?

Reply:
updated per your suggestion